lib/features/public_asset/data/network/public_asset_api.dart

```diff
@@ -29,6 +31,20 @@ class PublicAssetApi {
     return MapEntry(publicAsset.id!, patchObject);
   }
 
+  MapEntry<PublicAssetId, PatchObject> _toPartialPatchObjectMapEntry(
+    PublicAssetId publicAssetId,
+    UpdatingIdentityIds updatingIdentityIds,
+  ) {
+    assert(
+      updatingIdentityIds.values.where((value) => value == false).isEmpty,
+      'All updating identity id values must be true or null'
```

Comment:
Where is string?

Reply:
What do you mean?

Reply:
Why is this string there? Where did it come from?

Reply:
It is there for the developers only. As is says, that `PublicAsset/set update` request only accept `true` or `null` value.